zerodark-theme.el

```diff
@@ -121,6 +123,9 @@
                                                (propertize "RO " 'face 'bold))
                                            "")))
 
+(defvar zerodark-modeline-evil '(:eval (if evil-mode
+                                           (propertize (powerline-evil-tag) 'face 'bold) "")))
```

Comment:
Nope this wont fail. It's entirely dependant on `powerline-evil` so even if `evil-mode` is loaded and `powerline-evil` isn't nothing will happen. Same for the other way around if `powerline-evil` is loaded but no `evil-mode` then the checks for the current mode will just be the empty string so it won't fail and won't change the appearance. I just tested both.